op-deployer/pkg/deployer/pipeline/opchain.go

```diff
@@ -94,8 +94,15 @@ func DeployOPChain(ctx context.Context, env *Env, bundle ArtifactsBundle, intent
 		DelayedWETHPermissionlessGameProxyAddress: dco.DelayedWETHPermissionlessGameProxy,
 	})
 
-	currentBlock, _ := env.L1Client.BlockNumber(ctx)
-	block, _ := env.L1Client.BlockByNumber(ctx, big.NewInt(int64(currentBlock)))
+	currentBlock, err := env.L1Client.BlockNumber(ctx)
+	if err != nil {
+		return fmt.Errorf("failed to get current block number: %w", err)
+	}
+
+	block, err := env.L1Client.BlockByNumber(ctx, big.NewInt(int64(currentBlock)))
```

Comment:
You can just do `env.L1Client.BlockByNumber(ctx, nil)` to get the latest block.

Reply:
fixed in: https://github.com/ethereum-optimism/optimism/pull/12507/commits/c600c4e4af7dbd9da4f2c52056351f6198e74989